_includes/install_instructions/shell.html

```diff
@@ -0,0 +1,123 @@
+<div id="shell"> {% comment %} Start of 'shell' section. {% endcomment %}
```

Comment:
I think we don't need these `Start of ... section` / `End of ... section` comments